spotbugs-tests/src/test/java/edu/umd/cs/findbugs/ba/Issue758Test.java

```diff
@@ -18,21 +18,15 @@
  */
 package edu.umd.cs.findbugs.ba;
 
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.MatcherAssert.assertThat;
-
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
-
 import edu.umd.cs.findbugs.AbstractIntegrationTest;
-import edu.umd.cs.findbugs.SortedBugCollection;
 
 class Issue758Test extends AbstractIntegrationTest {
 
     @Test
     void testIssue() {
-        performAnalysis("ghIssues/Issue758.class");
-        SortedBugCollection bugCollection = (SortedBugCollection) getBugCollection();
-        assertThat(bugCollection.getErrors(), hasSize(0));
+        Assertions.assertDoesNotThrow(() -> performAnalysis("ghIssues/Issue758.class"));
```

Comment:
Original issue (https://github.com/spotbugs/spotbugs/issues/758) is about the analysis throwing exception.